app/javascript/components/App.js

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import PropTypes from "prop-types"
```

Comment:
- [ ] <b>REQUIRED:</b> This `PropType` is useless in this context. Kindly remove it.